support/ebpf/Makefile

```diff
@@ -16,6 +15,11 @@ else
 $(error Unsupported architecture: $(NATIVE_ARCH))
 endif
 
+# Get the current build directory (absolute path).
+BUILD_DIR := $(shell pwd)
```

Comment:
```suggestion
```

Reply:
In combination with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/560/files#r2181149647 should the idea be to rename `BUILD_DIR` to `CURDIR`? So far, `CURDIR` does not exist and with `BUILD_DIR` removed, the situation is unclear.

Reply:
`CURDIR` is built-into Make, no need to shell out and spawn a process.